### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by tdrkDev** [HandyGram/app#2] [Announcements]
Changelog:
* Added messages loading page
* New profile page between home and settings pages
* Fixed videos loading progress reporting
* Fixed chat history initial load
* Optimized UI for square screens
* Added ability to disable back buttons in app
* Fixed chats list lags (atleast I hope that they were fixed)
* Some other small changes

<hr /><em>This discussion was created from the release <a href='https://github.com/HandyGram/HandyGram/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by govinda9** [comment]
Closing the app with back button on GW5 leads to an unexpected error. When running the app again, it gets stuck in connecting. After rebooting the watch you can login to telegram again. Please fix!

**Reply by tdrkDev** [reply]
Hello, thanks for your report. I've discovered this bug some days ago and hope to fix it in 0.4.0 🤔

**Reply by itsmesomewherehere** [reply]
Same behavior on GW4.
 ̶I̶t̶ ̶i̶s̶ ̶a̶l̶s̶o̶ ̶u̶n̶c̶l̶e̶a̶r̶ ̶h̶o̶w̶ ̶t̶o̶ ̶r̶e̶s̶p̶o̶n̶d̶ ̶i̶n̶ ̶t̶h̶e̶ ̶c̶h̶a̶t̶.̶ ̶T̶h̶e̶ ̶m̶e̶n̶u̶ ̶T̶O̶D̶O̶ ̶r̶i̶s̶e̶s̶ ̶f̶r̶o̶m̶ ̶b̶e̶l̶o̶w̶ ̶a̶n̶d̶ ̶t̶h̶a̶t̶'̶s̶ ̶i̶t̶.̶ 
 after reading more carefully I realized that the functionality does not work yet )
![image](https://github.com/HandyGram/HandyGram/assets/133990122/1a617b44-66d2-49b9-81cf-00e4b16029b1)
![image](https://github.com/HandyGram/HandyGram/assets/133990122/345d97fb-1dd4-444b-8619-f8007259155b)
![image](https://github.com/HandyGram/HandyGram/assets/133990122/d0c8d209-486e-414d-bf68-adf29d0d8f4e)

**Comment by Lugib** [comment]
Hello,

great project.
Is it possible to write messages or send audio?
